charts/tidb-cluster/values.yaml

```diff
@@ -396,6 +397,7 @@ tidb:
   service:
     type: NodePort
     exposeStatus: true
+    # portName: "<the default name will be used>"
```

Comment:
can you replace `<the default name will be used>` with the actual default name used for tidb?